overrides.yaml

```diff
@@ -1,11 +0,0 @@
-override-remove:
- - moby-engine
- - zincati
```

Comment:
Let's leave these here? (I think it's nice to have this as an example of how to do overrides as well)